build/prettier.mk

```diff
@@ -0,0 +1,31 @@
+# ------------------------------------------------------------
+# Copyright 2023 The Radius Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#    
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------
+
+.PHONY: prettier-check prettier-format me prettier
+
+PRETTIER_VERSION := 3.3.3
+
+prettier-check:
```

Comment:
Please add documentation for these, use the other files as an example. When it's done you should see the new targets in the command line help printed by `make`.